electron-app/src/services/rotkehlchen-api.ts

```diff
@@ -282,6 +282,22 @@ export class RotkehlchenApi {
     });
   }
 
+  ping(): Promise<boolean> {
+    return new Promise<boolean>((resolve, reject) => {
```

Comment:
This wrapping is not necessary. It is mostly a leftover from the Zerorpc implementation. When I changed to axios it was not overly familiar with `axios` and keeping the wrapping was the fast way to move to the http api.
Since the axios api is promise based there is no need for that. 

You can look at an example here:

https://github.com/rotki/rotki/blob/ff4b654f28bedbd77159c6bfb383eaf294813611/electron-app/src/services/rotkehlchen-api.ts#L1286

Reply:
I changed this but I'm guessing to hold it until we decide what to do regarding the error discussion below?